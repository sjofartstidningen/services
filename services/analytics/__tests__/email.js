import { construct } from '../email';
import analyticsContext from '../../../test/mock-data/analytics-context.json';

jest.mock('../../../utils/logger.js');

describe('Module: email.construct', () => {
  it('should construct an email based on the given context', async () => {
    expect(await construct(analyticsContext)).toMatchSnapshot();
  });
});
