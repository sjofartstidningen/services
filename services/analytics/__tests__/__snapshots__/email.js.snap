// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Module: email.construct should construct an email based on the given context 1`] = `
"<!doctype html><html xmlns=\\"http://www.w3.org/1999/xhtml\\" xmlns:v=\\"urn:schemas-microsoft-com:vml\\" xmlns:o=\\"urn:schemas-microsoft-com:office:office\\"><head><title></title><!--[if !mso]><!-- --><meta http-equiv=\\"X-UA-Compatible\\" content=\\"IE=edge\\"><!--<![endif]--><meta http-equiv=\\"Content-Type\\" content=\\"text/html; charset=UTF-8\\"><meta name=\\"viewport\\" content=\\"width=device-width,initial-scale=1\\"><style type=\\"text/css\\">#outlook a { padding:0; }
          .ReadMsgBody { width:100%; }
          .ExternalClass { width:100%; }
          .ExternalClass * { line-height:100%; }
          body { margin:0;padding:0;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%; }
          table, td { border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt; }
          img { border:0;height:auto;line-height:100%; outline:none;text-decoration:none;-ms-interpolation-mode:bicubic; }
          p { display:block;margin:13px 0; }</style><!--[if !mso]><!--><style type=\\"text/css\\">@media only screen and (max-width:480px) {
            @-ms-viewport { width:320px; }
            @viewport { width:320px; }
          }</style><!--<![endif]--><!--[if mso]>
        <xml>
        <o:OfficeDocumentSettings>
          <o:AllowPNG/>
          <o:PixelsPerInch>96</o:PixelsPerInch>
        </o:OfficeDocumentSettings>
        </xml>
        <![endif]--><!--[if lte mso 11]>
        <style type=\\"text/css\\">
          .outlook-group-fix { width:100% !important; }
        </style>
        <![endif]--><!--[if !mso]><!--><link href=\\"https://fonts.googleapis.com/css?family=Ubuntu:300,400,500,700\\" rel=\\"stylesheet\\" type=\\"text/css\\"><style type=\\"text/css\\">@import url(https://fonts.googleapis.com/css?family=Ubuntu:300,400,500,700);</style><!--<![endif]--><style type=\\"text/css\\">@media only screen and (min-width:480px) {
        .mj-column-per-50 { width:50% !important; max-width: 50%; }
.mj-column-per-100 { width:100% !important; max-width: 100%; }
      }</style><style type=\\"text/css\\">@media only screen and (max-width:480px) {
      table.full-width-mobile { width: 100% !important; }
      td.full-width-mobile { width: auto !important; }
    }</style><style type=\\"text/css\\">a {
        font-family: Helvetica, Ubuntu, sans-serif;
        color: #0599e4;
        text-decoration: none;
      }

      a:hover {
        text-decoration: underline;
      }

      .thead {
        font-weight: normal;
        text-align: left;
      }

      .num {
        color: #9c9e9f;
        text-align: right;
        font-family: monospace;
      }</style></head><body style=\\"background-color:white;\\"><div style=\\"display:none;font-size:1px;color:#ffffff;line-height:1px;max-height:0px;max-width:0px;opacity:0;overflow:hidden;\\">Statistik för webb och nyhetsbrev vecka 1 – 2019</div><div style=\\"background-color:white;\\"><!--[if mso | IE]><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-bottom:0;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:300px;\\" ><![endif]--><div class=\\"mj-column-per-50 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"border-collapse:collapse;border-spacing:0px;\\"><tbody><tr><td style=\\"width:200px;\\"><img height=\\"auto\\" src=\\"https://www.sjofartstidningen.se/assets/2019/01/ST_logo.png\\" style=\\"border:0;display:block;outline:none;text-decoration:none;height:auto;width:100%;\\" width=\\"200\\"></td></tr></tbody></table></td></tr></table></div><!--[if mso | IE]></td><td class=\\"\\" style=\\"vertical-align:top;width:300px;\\" ><![endif]--><div class=\\"mj-column-per-50 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:12px;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:24px;line-height:1;text-align:left;color:#000000;\\">Statistik för webb<br>och nyhetsbrev</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-top:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:1;text-align:left;color:#9c9e9f;\\">vecka 1 – 2019</div></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-top:0px;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:600px;\\" ><![endif]--><div class=\\"mj-column-per-100 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><p style=\\"border-top:solid 1px #000000;font-size:1;margin:0px auto;width:100%;\\"></p><!--[if mso | IE]><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" style=\\"border-top:solid 1px #000000;font-size:1;margin:0px auto;width:550px;\\" role=\\"presentation\\" width=\\"550px\\" ><tr><td style=\\"height:0;line-height:0;\\"> &nbsp;
</td></tr></table><![endif]--></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-top:0px;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:600px;\\" ><![endif]--><div class=\\"mj-column-per-100 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:24px;text-align:left;color:#9c9e9f;\\">Här är statistik för Sjöfartstidningens hemsida samt nyhetsbrev under perioden 31 december – 6 januari 2019.</div></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-bottom:10px;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:600px;\\" ><![endif]--><div class=\\"mj-column-per-100 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:24px;line-height:1;text-align:left;color:#000000;\\">Räckvidd webb</div></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-top:0px;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:300px;\\" ><![endif]--><div class=\\"mj-column-per-50 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:1;text-align:left;color:#000000;\\">Senaste veckan:</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:12px;line-height:1;text-align:left;color:#9c9e9f;\\">Jämfört med föregående vecka</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><table cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\" border=\\"0\\" style=\\"cellspacing:0;color:#000000;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;table-layout:auto;width:100%;\\"><tr><th class=\\"thead\\">Besökare</th><td class=\\"num\\">11,562</td><td class=\\"num\\" style=\\"color: green;\\">23.6%</td></tr><tr><th class=\\"thead\\">Unika besökare</th><td class=\\"num\\">6,792</td><td class=\\"num\\" style=\\"color: green;\\">10.5%</td></tr><tr><th class=\\"thead\\">Sidvisningar</th><td class=\\"num\\">22,942</td><td class=\\"num\\" style=\\"color: green;\\">24.3%</td></tr></table></td></tr></table></div><!--[if mso | IE]></td><td class=\\"\\" style=\\"vertical-align:top;width:300px;\\" ><![endif]--><div class=\\"mj-column-per-50 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:1;text-align:left;color:#000000;\\">Senaste månaden:</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:12px;line-height:1;text-align:left;color:#9c9e9f;\\">Jämfört med samma månad förra året</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><table cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\" border=\\"0\\" style=\\"cellspacing:0;color:#000000;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;table-layout:auto;width:100%;\\"><tr><th class=\\"thead\\">Besökare</th><td class=\\"num\\">62,325</td><td class=\\"num\\" style=\\"color: red;\\">-9.6%</td></tr><tr><th class=\\"thead\\">Unika besökare</th><td class=\\"num\\">25,759</td><td class=\\"num\\" style=\\"color: red;\\">-3.7%</td></tr><tr><th class=\\"thead\\">Sidvisningar</th><td class=\\"num\\">128,667</td><td class=\\"num\\" style=\\"color: red;\\">-5.2%</td></tr></table></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:300px;\\" ><![endif]--><div class=\\"mj-column-per-50 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:1;text-align:left;color:#000000;\\">Mest lästa artiklarna:</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><table cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\" border=\\"0\\" style=\\"cellspacing:0;color:#000000;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;table-layout:auto;width:100%;\\"><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/rekordhog-sjo-och-installda-turer/\\">Rekordhög sjö och inställda...</a></th><td class=\\"num\\">1,368</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/kbv-ser-bemanningen/\\">KBV ser över bemanningen</a></th><td class=\\"num\\">1,044</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/tyskt-klartecken-tunnelbygge/\\">Tyskt klartecken för tunnelb...</a></th><td class=\\"num\\">772</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/bloggar/utkiks-bloggen/ett-svinaktigt-2019-stundar/\\">Ett svinaktigt 2019 stundar</a></th><td class=\\"num\\">513</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/sjofartsverket-foreslar-utokat-isbrytarsamarbete/\\">Föreslår utökat isbrytarsam...</a></th><td class=\\"num\\">505</td></tr></table></td></tr></table></div><!--[if mso | IE]></td><td class=\\"\\" style=\\"vertical-align:top;width:300px;\\" ><![endif]--><div class=\\"mj-column-per-50 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:1;text-align:left;color:#000000;\\">Mest lästa jobben:</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><table cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\" border=\\"0\\" style=\\"cellspacing:0;color:#000000;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;table-layout:auto;width:100%;\\"><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/jobb/fartygsingenjor-3/\\">Fartygsingenjör</a></th><td class=\\"num\\">104</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/jobb/befraktare-samt-tva-fartygsoperatorer/\\">Befraktare samt två fartygs...</a></th><td class=\\"num\\">101</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/jobb/saljareaffarsutvecklare-2/\\">Säljare/Affärsutvecklare</a></th><td class=\\"num\\">85</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/jobb/teknisk-chef-8/\\">Teknisk chef</a></th><td class=\\"num\\">85</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/jobb/overstyrman-till-astrea/\\">Överstyrman till ”Astrea”</a></th><td class=\\"num\\">69</td></tr></table></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-top:0px;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:600px;\\" ><![endif]--><div class=\\"mj-column-per-100 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><p style=\\"border-top:solid 1px #000000;font-size:1;margin:0px auto;width:100%;\\"></p><!--[if mso | IE]><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" style=\\"border-top:solid 1px #000000;font-size:1;margin:0px auto;width:550px;\\" role=\\"presentation\\" width=\\"550px\\" ><tr><td style=\\"height:0;line-height:0;\\"> &nbsp;
</td></tr></table><![endif]--></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-bottom:10px;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:600px;\\" ><![endif]--><div class=\\"mj-column-per-100 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"vertical-align:top;\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:24px;line-height:1;text-align:left;color:#000000;\\">Räckvidd nyhetsbrev</div></td></tr></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\" ><tr><td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\"><![endif]--><div style=\\"Margin:0px auto;max-width:600px;\\"><table align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"><tbody><tr><td style=\\"direction:ltr;font-size:0px;padding:20px 0;padding-top:0px;text-align:center;vertical-align:top;\\"><!--[if mso | IE]><table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td class=\\"\\" style=\\"vertical-align:top;width:600px;\\" ><![endif]--><div class=\\"mj-column-per-100 outlook-group-fix\\" style=\\"font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"><tbody><tr><td style=\\"vertical-align:top;padding-bottom:20px;\\"><table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:19.2px;text-align:left;color:#000000;\\">Sjöfartstidningens nyhetsbrev 190103</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:12px;line-height:1;text-align:left;color:#9c9e9f;\\">Föreslår utökat isbrytarsamarbete</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><table cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\" border=\\"0\\" style=\\"cellspacing:0;color:#000000;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;table-layout:auto;width:100%;\\"><tr><th class=\\"thead\\">Mottagare</th><td class=\\"num\\">8,626</td></tr><tr><th class=\\"thead\\">Andel som öppnat</th><td class=\\"num\\">35.3%</td></tr><tr><th class=\\"thead\\">Andel som klickat</th><td class=\\"num\\">13.5%</td></tr></table></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;padding-bottom:0;word-break:break-word;\\"><div style=\\"font-family:Helvetica, Ubuntu, sans-serif;font-size:16px;line-height:1;text-align:left;color:#000000;\\">Mest klickade artiklarna:</div></td></tr><tr><td align=\\"left\\" style=\\"font-size:0px;padding:10px 25px;word-break:break-word;\\"><table cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\" border=\\"0\\" style=\\"cellspacing:0;color:#000000;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;table-layout:auto;width:100%;\\"><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/bloggar/utkiks-bloggen/ett-svinaktigt-2019-stundar/?utm_source&#x3D;Sj%C3%B6fartstidningens+nyhetsbrev&amp;utm_campaign&#x3D;9a9032bcce-EMAIL_CAMPAIGN_2019_01_03_03_08&amp;utm_medium&#x3D;email&amp;utm_term&#x3D;0_eaa9b84478-9a9032bcce-[LIST_EMAIL_ID]\\">/bloggar/utkiks-bloggen/ett-...</a></th><td class=\\"num\\">385</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/kbv-ser-bemanningen/?utm_source&#x3D;Sj%C3%B6fartstidningens+nyhetsbrev&amp;utm_campaign&#x3D;9a9032bcce-EMAIL_CAMPAIGN_2019_01_03_03_08&amp;utm_medium&#x3D;email&amp;utm_term&#x3D;0_eaa9b84478-9a9032bcce-[LIST_EMAIL_ID]\\">/kbv-ser-bemanningen/?utm...</a></th><td class=\\"num\\">337</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/rekordhog-sjo-och-installda-turer/?utm_source&#x3D;Sj%C3%B6fartstidningens+nyhetsbrev&amp;utm_campaign&#x3D;9a9032bcce-EMAIL_CAMPAIGN_2019_01_03_03_08&amp;utm_medium&#x3D;email&amp;utm_term&#x3D;0_eaa9b84478-9a9032bcce-[LIST_EMAIL_ID]\\">/rekordhog-sjo-och-installda...</a></th><td class=\\"num\\">302</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/sjofartsverket-foreslar-utokat-isbrytarsamarbete/?utm_source&#x3D;Sj%C3%B6fartstidningens+nyhetsbrev&amp;utm_campaign&#x3D;9a9032bcce-EMAIL_CAMPAIGN_2019_01_03_03_08&amp;utm_medium&#x3D;email&amp;utm_term&#x3D;0_eaa9b84478-9a9032bcce-[LIST_EMAIL_ID]\\">/sjofartsverket-foreslar-utok...</a></th><td class=\\"num\\">266</td></tr><tr><th class=\\"thead\\"><a href=\\"https://www.sjofartstidningen.se/tyskt-klartecken-tunnelbygge/?utm_source&#x3D;Sj%C3%B6fartstidningens+nyhetsbrev&amp;utm_campaign&#x3D;9a9032bcce-EMAIL_CAMPAIGN_2019_01_03_03_08&amp;utm_medium&#x3D;email&amp;utm_term&#x3D;0_eaa9b84478-9a9032bcce-[LIST_EMAIL_ID]\\">/tyskt-klartecken-tunnelbygg...</a></th><td class=\\"num\\">262</td></tr></table></td></tr></table></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><![endif]--></td></tr></tbody></table></div><!--[if mso | IE]></td></tr></table><![endif]--></div></body></html>"
`;
